### ISSUE TITLE
hi please help me!!!

### ISSUE THREAD

**Issue by HEZARAKOR001** [Nickelza/one-piece-wanted-poster-generator-website#6]
hello i saw your onepiece wanted poster maker and i absoultly like it i want to create the same page in my django project plzz send me the frontend or backend or tell me a way to do so plzz your help will be really appriciatable(sorry my english is not good)
plzz help me thanks

**Comment by The8th-Hokage** [comment]
Were u by chance from Telegram? https://t.me/bountysystem join there at Telegram for support

**Comment by Nickelza** [comment]
> hello i saw your onepiece wanted poster maker and i absoultly like it i want to create the same page in my django project plzz send me the frontend or backend or tell me a way to do so plzz your help will be really appriciatable(sorry my english is not good) plzz help me thanks

@HEZARAKOR001 just happened to see this issue now, I guess you have probably resolved but I'll leave the answer in case anyone else should have the same question:

The "backend" is merely the package [One Piece Wanted Poster](https://github.com/Nickelza/one-piece-wanted-poster) which has a detailed guide on how to use

**Comment by Nickelza** [comment]
> Were u by chance from Telegram? https://t.me/bountysystem join there at Telegram for support

@The8th-Hokage Yes, I am also the creator of [One Piece Group Bot](https://github.com/Nickelza/one-piece-group-bot), of which the official group is https://t.me/bountysystem

**Comment by The8th-Hokage** [comment]
I know that you are the creator of both. I am not the one who wrote the above comments.
